feat:  nydusify can dump metrics created by converter

Desiki-high opened the pull request:
### We are introducing a new arg called "output-json" that will allow us to save the metrics generated by convert. This will enable us to successfully implement https://github.com/dragonflyoss/image-service/issues/1263 through nydusify. 
For additional information, please see https://github.com/goharbor/acceleration-service/pull/133.

Desiki-high committed:
nydusify: update acceleration-service version

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

Desiki-high commented:
The output json like this:
![image](https://github.com/dragonflyoss/image-service/assets/59167816/8f1c2a1a-995c-43f3-9ca0-3429ae116757)

Desiki-high commented:
/retest

imeoer commented on contrib/nydusify/cmd/nydusify.go:
```diff
@@ -409,6 +409,12 @@ func main() {
+				&cli.BoolFlag{
+					Name:    "output-json",
+					Value:   false,
+					Usage:   "Dump the metric colledted in convert to local json file: output.json",
```
Enable saving the metrics collected during conversion in JSON file output.json.

Desiki-high replied:
fixed

Desiki-high committed:
nydusify: add new option output-json

During convert, we can collect the metric: image size and convert time, etc.
Nydusify can dump the metric to local file if user needs.

Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

Desiki-high commented on contrib/nydusify/pkg/converter/metric.go:
```diff
@@ -0,0 +1,29 @@
+
+func dumpMetric(metric *converter.Metric) error {
+
+	file, err := os.Create("output.json")
```
fixed

Desiki-high commented on contrib/nydusify/pkg/converter/metric.go:
```diff
@@ -0,0 +1,29 @@
+	if err != nil {
+		return errors.Wrap(err, "Error encoding JSON from metric")
+	}
+	return nil
```
fixed
